extensions/wikia/PortabilityDashboard/PortabilityDashboardModel.class.php

```diff
@@ -47,7 +47,8 @@ private function getRowList() {
 						'traffic' => $row->traffic,
 						'migrationImpact' => $row->migration_impact,
 						'typelessTemplatesCount' => $row->typeless,
-						'customInfoboxesCount' => $row->custom_infoboxes
+						'customInfoboxesCount' => $row->custom_infoboxes,
+						'lastUpdated' => date('d-m-Y', strtotime($row->updated))
```

Comment:
I've not contacted anyone, but it should give them exact information on the last update per wiki.